itest/lnd_open_channel_test.go

```diff
@@ -877,6 +877,114 @@ func testFundingExpiryBlocksOnPending(ht *lntest.HarnessTest) {
 	ht.MineBlocksAndAssertNumTxes(1, 1)
 }
 
+// testPendingChannelConfirmationUntilActive verifies the value for the rpc
+// filed ConfirmationUntilActive decreases as expected as soon as blocks are
```

Comment:
filed ? did you mean field ? => rpc field `ConfirmationUntilActive` updates correctly as soon as ..